### ISSUE TITLE
Replace DarkTurquoise with ForestGreen across all site

### ISSUE THREAD

**Issue by panchovm** [creativecommons/caselaw#62]
## Problem
Since we defined to start with AA accessibility level, the `DarkTurquoise` color does not match at all the contrast ratio for small and large text sizes. This is a brand concern that we have been discussing and we will come with the final brand solution after designing the .org site.

In parallel, I have been trying to adjust the `DarkTurquoise`, but its nature is low-contrast and bright. Adjusting this color means changing it completely.

I assign myself a task (#48) to work on the interaction color for the `DarkTurquoise`. But as I said above, the issue is bigger than the use on this site.

## Solution
We can replace the `DarkTurquoise` with a different brand color. And based on the AA level and contrast ratio required, the `ForestGreen` seems to match better. For `ForestGreen`, we do have an interaction color called `BrighterForestGreen`. So all `hover` states can use it without problem.

I duplicated the home page replacing the `DarkTurquoise` with `ForestGreen`. So please @krysal @kgodey @TimidRobot comment this ticket to see your thoughts about this change. The accessibility is a crucial dimension of Vocabulary and, unfortunately, the conversation about adjusting brand colors came after starting this project.

### Mockup sample
<img width="566" alt="Screen Shot 2020-08-13 at 09 36 56" src="https://user-images.githubusercontent.com/895819/90141021-9414db00-dd48-11ea-9686-1ca8c63e0acd.png">

**Comment by kgodey** [comment]
I am fine with it.

**Comment by krysal** [comment]
Seems like a right move, I'm fine with it too.

**Comment by Aakash2408** [comment]
Is this issue still open?

**Comment by krysal** [comment]
@Aakash2408 Yes, it is a change pending.

**Comment by Aakash2408** [comment]
@krysal Can you assign this to me ?

**Comment by Aakash2408** [comment]
> @Aakash2408 Yes, it is a change pending.

I guess the changes need to be made in main.css which is under assets folder which I can't find :(
![Screenshot from 2020-08-17 08-28-11](https://user-images.githubusercontent.com/47134609/90353364-9dad8400-e063-11ea-8441-8dfbb0f74325.png)

Am I right @krysal ?

**Comment by Aakash2408** [comment]
Thanks for the information 😃.

> @Aakash2408 I guess you cloned the default branch (`master`) as is the natural way. The thing is that this projects is under a reimplementation with Django, a Python's framework, and integrating [Vocabulary](https://cc-vocabulary.netlify.com/), the CC's design system, so the version discussed here is in the `develop` branch and the file to edit is `legal_db/static/styles.scss`, sorry for not warning of this before, hope this helps you get started.
> 
> Thanks for showing interest, your help is welcome!

Thanks for the information!

**Comment by Aakash2408** [comment]
@kgodey  @krysal @panchovm  I am replacing the dark turquoise with forest green (RGB(34,139,34)).Kindly check and confirm the RGB values.SO that I can make the changes.

**Comment by panchovm** [comment]
Hi @Aakash2408. The RGB value of `Forest Green` is `RGB(4,166,53)` and the Hex code `#04A635`

**Comment by Aakash2408** [comment]
> Hi @Aakash2408. The RGB value of `Forest Green` is `RGB(4,166,53)` and the Hex code `#04A635`

ok Thanks! I will just change it and am going to make another pull request there are some issues with this pull request.

**Comment by Aakash2408** [comment]
Hey @krysal,  Need any more changes?

**Comment by krysal** [comment]
This is done. Closing.
